graph2tac/loader/predict_server.py

```diff
@@ -426,63 +564,46 @@ def prediction_loop(predict_server: PredictServer, context: GlobalContextMessage
     log_cnts.session_idx += 1
     log_cnts.thm_idx = -1
 
-    msg = next(message_iterator, None)
-    while msg is not None:
-        if isinstance(msg, ProofState):
-            log_cnts.thm_idx += 1
-            log_cnts.thm_annotation = context.log_annotation
-
-            # FIXME: Implement incremental updating of the context
-            with predict_server.coq_context(context):
-
-                log_cnts.msg_idx = -1
-                log_cnts.t_predict = 0.0
-                log_cnts.n_predict = 0
-                log_cnts.max_t_predict = 0.0
-                log_cnts.argmax_t_predict = -1
-                log_cnts.t_coq = 0.0
-                log_cnts.n_coq = 0
-                t0_coq = time.time()
```

Comment:
Looks like the logging counters are broken?

Reply:
Oh, you are right, thanks for the remainder, I was planning to look at them later. I will add this to the TODO list at the top.